src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public DataProcessingException(String message) {
+        super(message);
+    }
```

Comment:
it looks like you don't use this constructor, remove it if it is not used anywhere